Documentation/devel/stage1-implementors-guide.md

```diff
@@ -135,3 +134,26 @@ Examples
     ]
 }
 ```
+
+## Filesystem Layout Assumptions
+
+The following paths are reserved for the stage1 image, and they will be created during stage0.
+Users **SHOULD NOT** create or use these paths when making a stage1 image.
```

Comment:
@yifan-gu sorry my comment here seems to have been lost. I'm a little wary of the word "Users" as I don't want to mislead anyone - I suggest a slight tweak:
> When creating a stage1 image, developers **SHOULD NOT** create or use these paths in the image's filesystem